packages/core/next.config.js

```diff
@@ -18,6 +18,9 @@ const nextConfig = {
   sassOptions: {
     additionalData: `@import "src/customizations/styles/custom-mixins.scss";`,
   },
+  experimental: {
+    scrollRestoration: true,
```

Comment:
We can remove this when migrating to [Next.js 13](https://twitter.com/felixmokross/status/1654215560677277701)

Reply:
Add a comment on the code pls!